integration-tests/ahm/src/proxy/basic_still_works.rs

```diff
@@ -272,12 +261,12 @@ impl ProxyBasicWorks {
 	) -> <AssetHubRuntime as pallet_balances::Config>::Balance {
 		let ed = <AssetHubRuntime as pallet_balances::Config>::ExistentialDeposit::get();
 		let _ = pallet_balances::Pallet::<AssetHubRuntime>::deposit_creating(
-			&delegatee.clone().into(),
-			ed * 10000000,
+			&delegatee.clone(),
+			ed * 100_000_000_000,
```

Comment:
used to be `10_000_000`

Reply:
Yea it did not work anymore with the old value after updating the snapshot. I have not checked why yet, but some of the accounts had less balance than before. So maybe the old amount was not enough anymore to make the test work.